pkg/eval/json_evaluator_test.go

```diff
@@ -1065,3 +1066,132 @@ func TestState_Evaluator(t *testing.T) {
 		})
 	}
 }
+
+func TestFlagStateSafeForConcurrentReadWrites(t *testing.T) {
```

Comment:
This test fails with concurrent r/w of map error without these changes

Reply:
Thanks for finding/fixing/testing!